pkg/backup/item_backupper.go

```diff
@@ -310,8 +317,32 @@ func (ib *itemBackupper) executeActions(
 			continue
 		}
 		log.Info("Executing custom action")
+		if ib.backupRequest.ResPolicies != nil && groupResource.String() == "persistentvolumeclaims" && action.Name() == "velero.io/csi-pvc-backupper" {
+			pvc := corev1api.PersistentVolumeClaim{}
+			if err := runtime.DefaultUnstructuredConverter.FromUnstructured(obj.UnstructuredContent(), &pvc); err != nil {
+				return nil, itemFiles, errors.WithStack(err)
+			}
+
+			pvName := pvc.Spec.VolumeName
+			if pvName == "" {
+				return nil, itemFiles, errors.Errorf("PVC has no volume backing this claim")
+			}
+
+			pv := &corev1api.PersistentVolume{}
+			if err := ib.kbClient.Get(context.Background(), kbClient.ObjectKey{Name: pvName}, pv); err != nil {
+				return nil, itemFiles, errors.WithStack(err)
+			}
 
+			volume := resourcepolicies.StructuredVolume{}
+			volume.ParsePV(pv)
+			act := ib.backupRequest.ResPolicies.Match(&volume)
+			if act != nil && act.Type == resourcepolicies.Skip {
+				log.Infof("skip snapshot of pv %s for the matched resource policies", pvName)
+				return nil, itemFiles, nil
```

Comment:
We should continue to run other BIAs rather than return

Reply:
yeah, Updated